### PR TITLE
Add centos 7 installation howto

### LINKED ISSUE #1834: Add a HowTo for CentOS7

### PR DESCRIPTION
**Pull request by plettich**
Fixes #1834

### CODE REVIEW

**Comment by plettich** [comment]
Maybe i should remove the leading ``$``  to make copy/paste easier?

File: doc/installation/centos.rst [hunk 1]
```diff
@@ -4,10 +4,168 @@
...
+    (privacyidea) $ echo "PI_PEPPER = '$PEPPER'" >> /etc/privacyidea/pi.cfg
+    (privacyidea) $ SECRET="$(tr -dc A-Za-z0-9_ </dev/urandom | head -c24)"
+    (privacyidea) $ echo "SECRET_KEY = '$SECRET'" >> /etc/privacyidea/pi.cfg
+
+From now on the ``pi-manage``-tool can be used to configure and manage the privacyIDEA server::
+
+    (privacyidea) $ pi-manage create_enckey  # encryption key for the database
+    (privacyidea) $ pi-manage create_audit_keys  # key for verification of audit log entries
+    (privacyidea) $ pi-manage createdb  # create the database structure
+    (privacyidea) $ pi-manage db stamp head -d lib/privacyidea/migrations/  # stamp the db
```
Comment (uname-hub) [inline doc/installation/centos.rst]: Instead of lib/privacyidea/... the full path /opt/.... should be used.

Comment (plettich) [reply doc/installation/centos.rst]: fixed in e3393257

File: doc/installation/centos.rst [hunk 2]
```diff
@@ -4,10 +4,180 @@
...
 
+Step-by-Step installation on CentOS
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
 .. index:: CentOS, Red Hat, RHEL
 
-There is a detailed Howto [#howto]_ for installing privacyIDEA with FreeRADIUS
-3 on CentOS 7 using a python virtual environment.
+In this chapter we describe a way to install privacyIDEA on CentOS 7 based on the
+installation via :ref:`pip_install`. It follows the
```
Comment (timosturm) [inline doc/installation/centos.rst]: ":ref:" does not seem to lead anywhere.

File: doc/installation/centos.rst [hunk 3]
```diff
@@ -4,10 +4,180 @@
...
+    PI_LOGFILE = '/var/log/privacyidea/privacyidea.log'
+    PI_LOGLEVEL = logging.INFO
+
+Make sure the configuration file is not world readable:
+
+.. code-block:: bash
+
+    (privacyidea)$ chmod 640 /etc/privacyidea/pi.cfg
+
+More information on the configuration parameters can be found in :ref:`cfgfile`.
```
Comment (timosturm) [inline doc/installation/centos.rst]: This ":ref:" just jumps to the start of the page.

File: doc/installation/centos.rst (hunk 2 above)
Comment (plettich) [reply doc/installation/centos.rst]: The sphinx-tool is quite good at resolving references. When i build the documentation, this reference leads to [Python package index](https://github.com/privacyidea/privacyidea/blob/master/doc/installation/pip.rst) page.

File: doc/installation/centos.rst (hunk 3 above)
Comment (plettich) [reply doc/installation/centos.rst]: Same here, it takes me to the inifile page.

**Pull request merged**